scripts/example-mc.jl

```diff
@@ -0,0 +1,77 @@
+# Define parameters for MCMC simulation
+L = square_supercell_dimensions[1]
+M = square_supercell_dimensions[2]
+Ns = [4]
+adsorption_energy = -0.04
+nn_energy = -0.01
+nnn_energy = -0.0025
+temperatures = 200:-1:1
+num_equilibration_steps = 10_000
+num_sampling_steps = 10_000
+random_seed = 1234
+k_B = 8.617_333_262e-5
+
+# Define file paths for data
+parameters = "mc_square_L
+(M)_adsorption
+(nn_energy)_nnn
+(square_cutoff_radii[1])_$(square_cutoff_radii[2])"
+
+
+df_samples = DataFrame(N = Int[], T = Float64[], E = Float64[], Cv = Float64[], acceptance_rate = Float64[])
+
+# Perform the Monte Carlo simulation
+for N in Ns
+    println("N = $N -------------------")
+
+    # Initialize the lattice
```

Comment:
Do we have any convenience functions for lattice initialization yet?

Reply:
Getting there! Adding more!